src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -214,8 +216,7 @@ class ItemPage extends React.Component {
 
         <div className="nypl-full-width-wrapper">
           <div className="nypl-row">
-            <div className="nypl-column-three-quarters nypl-column-offset-one">
-            </div>
+            <Search sortBy={sortBy} />
```

Comment:
The search component itself has those classes. If you inspect the element on the page, it's rendering with those classes.